app/services/SchemesOverviewService.scala

```diff
@@ -43,31 +40,29 @@ class SchemesOverviewService @Inject()(appConfig: FrontendAppConfig,
   def getTiles(psaId: String)(implicit request: OptionalDataRequest[AnyContent], hc: HeaderCarrier, messages: Messages): Future[Seq[CardViewModel]] =
     for {
       invitationLink <- invitationsLink
-      adminHtml <- retrievePenaltiesUrlPartial
       schemeHtml <- frontendConnector.retrieveSchemeUrlsPartial
     } yield {
       Seq(
-        adminCard(invitationLink, psaId, adminHtml),
+        adminCard(invitationLink, psaId),
         schemeCard(schemeHtml)
       )
     }
 
+  def retrievePenaltiesUrlPartial[A](implicit request: Request[A], ec: ExecutionContext): Future[Html] =
+    if(appConfig.isFSEnabled) {
```

Comment:
Is this toggle still in play or can we remove it?

Reply:
I think it can be removed. I don't know how many refs there are to it throughout all repos tho, be worth doing a check

Reply:
Raise a ticket to look at that

Reply:
ok will do